### DISCUSSION TITLE
channel_list() does not exist

### DISCUSSION THREAD

**Discussion by markosnarinian** [supersaiyanmode/PyWebOSTV#53] [Q&A]
This is my code:
```
tv = WebOSClient('192.168.1.13')
tv.connect()
for status in tv.register(store):
    if status == WebOSClient.PROMPTED:
        print("Please accept the connect on the TV!")
    elif status == WebOSClient.REGISTERED:
        print("Registration successful!\n")

tv_control = TvControl(tv)

tv_control.channel_list()
```

When I run my script I get the following error:
```
File "/home/pi/.local/lib/python3.7/site-packages/pywebostv/controls.py", line 91, in __getattr__
    raise AttributeError(name)
AttributeError: channel_list
```

**Comment by supersaiyanmode** [comment]
I haven't tried running your code snippet, but could you please update the code in the following way and post your results?

    tv_control = TvControl(tv)
    print(tv_control.COMMANDS["channel_list"])

.. you should see a dict output instead of a KeyError or AttributeError. If you don't have any problems here -- then can you please go to the place where this library is installed, and insert a tiny [debug log line before this point](https://github.com/supersaiyanmode/PyWebOSTV/blob/master/pywebostv/controls.py#L70) and find out why the `if` condition is failing? The expression should evaluate to `True` in your case.
